packages/web3-core/README.md

```diff
@@ -24,7 +23,7 @@ var CoolLib = function CoolLib() {
 
     // sets _requestmanager and adds basic functions
     core.packageInit(this, arguments);
```

Comment:
these are all internal methods though? we shouldn't need to type them out only should be doing the ones which affect the public interface...? It's a waste of time otherwise.

Reply:
i also did bring in the types we did on 2.x into here so please review them.